docs/content/operator/validator-node/configuration.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Validator Node Configuration
+description: Prepare the configuration file for the IOTA validator node.
+tags: [node-operation]
+teams:
+  - iotaledger/node
+---
+
+The IOTA validator node configuration file is a YAML file (usually named `validator.yaml`) that you can use to configure
+your `iota-node`.
+ To simplify the setup process, you can use a validator node configuration file [template](https://github.com/iotaledger/iota/raw/refs/heads/develop/nre/config/validator.yaml) and modify it to your needs.
```

Comment:
```suggestion
 To simplify the setup process, you can use a validator node configuration file [template](https://github.com/iotaledger/iota/raw/refs/heads/develop/setups/validator/systemd/validator.yaml) and modify it to your needs.
```

Reply:
Fixed in ede15b84a110e29ed56b96fd6ceade6336c84587